docs/63_dandi_infrastructure.md

```diff
@@ -0,0 +1,203 @@
+
+### Netlify
+
+Although Netlify prescribes mapping of Netlify-issued DNS records directly, DANDI Infrastructure relies on mapping Netlify's Load Balancer IP to the respective A Name Record in AWS Route 53, 
+as prescribed in [Netlify's docs here](https://docs.netlify.com/domains-https/custom-domains/configure-external-dns/#configure-an-apex-domain)
+
+```
+resource "aws_route53_record" "gui" {
+  zone_id = aws_route53_zone.dandi.zone_id
+  name    = "" # apex
+  type    = "A"
+  ttl     = "300"
+  records = ["75.2.60.5"] # Netlify's load balancer, which will proxy to our app
+}
+```
+
+Note the code blob above, as referenced from [DANDI Infrastructure domain.tf Terraform template](https://github.com/dandi/dandi-infrastructure/blob/master/terraform/domain.tf#L13).
+
+### AWS Route 53 and ACM
+
+A manual step is necessary to set up an SSL Certificate for your DNS records throughout your DANDI clone. 
+
+Proceed to AWS `Certificate Manager`. Begin by requesting a certificate -- **Note: Ensure you are in the same region as the default you have provided in your Terraform `main.tf` template.
+
+<br/><br/>
+<img
+src="../img/aws_acm_dashboard.jpg"
+alt="aws_acm_dashboard"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Next, request a `Public Certificate`
+
+<br/><br/>
+<img
+src="../img/public_cert.jpg"
+alt="public_cert"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+When setting up the certificate, provide a value of `*.<your_domain_name>` -- the goal being that the `*` will serve as a wildcard for both your API and UI DNS records
+
+<br/><br/>
+<img
+src="../img/valid_domain_acm.jpg"
+alt="valid_domain_acm"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Lastly, validate your certificate via DNS Records. This can be done by using the CNAME values 
+expressed by the certificate and linking them as records in your DNS Hosted Zone.
+
+### Heroku Dyno Sizes
+
+`dandi-infrastructure` defines "dyno" (a.k.a process) sizes for each service being run. For specific reference,
+[see here in api.tf](https://github.com/dandi/dandi-infrastructure/blob/master/terraform/api.tf#L14-L18).
+
+While your DANDI Archive clone may differ in traffic and activity, the defaults set in `dandi-infrastructure` rarely
+exceed 75% usage.
+
+Keep in mind the different [pricing structures](https://www.heroku.com/pricing) that come with choosing different Heroku dyno sizes
+
+### Heroku Add-Ons
+
+In addition to the Heroku 'dynos' that are added for compute, multiple Heroku 'add-ons' are included
+
+- [CloudAMQP](https://elements.heroku.com/addons/cloudamqp) -- **Use Case**: Message Broker
+- [Postgres](https://elements.heroku.com/addons/heroku-postgresql) -- **Use Case**: Database
+- [Papertrail](https://elements.heroku.com/addons/papertrail) -- **Use Case**: Log Management
+
```

Comment:
Should we move these to the `Understanding the DANDI Infrastructure` section in this page?

Reply:
I believe they are already in that section? Unless you had something else in mind?